### ISSUE TITLE
GlobalAveragePooling2D causes the inconsistency of output between frameworks

### ISSUE THREAD

**Issue by Crystal-yyy** [keras-team/keras#16504]
**System information.**
Ubuntu 20.04.2 LTS (GNU/Linux 5.4.0-74-generic x86_64)
TensorFlow 2.0.0 installed by anaconda
gcc (Ubuntu 9.3.0-17ubuntu1~20.04) 9.3.0
NVIDIA-SMI 460.80 Driver Version: 460.80 CUDA Version: 11.2

**Describe the problem.**
Operators with Sum and Avg often cause inconsistency between frameworks. And for this inconsistency, only changing the input may cause its nature to change.

**For example.**
structure：GlobalAvgPool2d + Flatten + Dense + Softmax
dataset：mnist 
batchsize：25
When batchsize is 25， it will appear as Tensorflow Inconsistency.
（Inconsistency: When testing the three frameworks, the inconsistency here refers to comparing the output of the GlobalAvgPool2d layer with the other two frameworks and computing their absolute values.）
